js/app.js

```diff
@@ -1 +1,38 @@
-(function() {})()
+$(".dropdown-menu li a").click(function(){
```

Comment:
Or better, when I last used jquery (a long time ago) you would put everything in something like `$(document).ready(() => {})` (I think)